java/serve/src/main/java/io/ray/serve/config/RayServeConfig.java

```diff
@@ -0,0 +1,13 @@
+package io.ray.serve.config;
+
+public class RayServeConfig {
+
+  public static final String PROXY_CLASS = "ray.serve.proxy.class";
+
+  public static final String METRICS_ENABLED = "ray.serve.metrics.enabled";
+
+  public static final String LONG_POOL_CLIENT_ENABLED = "ray.serve.long.poll.client.enabled";
+
+  /** The polling interval of long poll thread. Unit: second. */
+  public static final String LONG_POOL_CLIENT_INTERVAL = "ray.serve.long.poll.client.interval";
```

Comment:
Same general feedback as [this comment](https://github.com/ray-project/ray/pull/22726/files#r893937193).

Reply:
Resolved.